default.yaml

```diff
@@ -146,8 +146,8 @@ onChainWallet:
 # TODO
 # carrierRegexFilter: ""
 
-helmet:
-  disableContentPolicy: false
+apollo:
+  playground: false
```

Comment:
I would have this on by default. wdyt?

we can disable it on the charts so that it's by default off in production

Reply:
if you decide to set to true by default please add additional validation for tests in writeSDLFile to avoid open handles

Reply:
I'm actually in favor of removing defaults all together. Every environment should have its own configurations. 

Having defaults carries the risk of one environment using a default value when it shouldn't. Setting things closer to production is a safer bet for now (it's still risky too).

Reply:
I like having explicit settings per environment to a certain extent we already have this via the `charts` + `deployments` repos the only thing that needs specifying in this repo are the dev settings. How would you make this explicit in the file structure so that the workflow is streamlined?